plugins/orchestrator-backend/src/service/router.ts

```diff
@@ -741,7 +741,7 @@ function setupInternalRoutes(
       }
 
       if (!definition.dataInputSchema) {
-        res.status(200).json(res);
+        res.status(200).json({});
```

Comment:
I added a comment but i think something went wrong in GitHub... 
My comment was "Is it ok to have `200`? Is this a successful response?"

Reply:
Ok I see the PR has been closed and reopen, and that @batzionb replied.